src/components/BugReportView.tsx

```diff
@@ -26,15 +29,37 @@ export const escapePII = (text: string, filterFields: string[]): string => {
         ;
 };
 
-const getBugReportBody = (filterFields: string[]): string => {
+const deviceInfo = () => {
+    const brand = DeviceInfo.getBrand();
+    const deviceID = DeviceInfo.getDeviceId();
```

Comment:
handling user/device ids is mentioned in the latest android beta blogpost, also references some best practices. It may be related:
https://android-developers.googleblog.com/2019/03/introducing-android-q-beta.html

Reply:
Here the DeviceId refers to the model of a maker, e.g. iPhone 8 or Pixel XL3.